app/ide-desktop/lib/dashboard/src/providers/RemoteBackendProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import RemoteBackend from '#/services/RemoteBackend'
+
+import HttpClient from '#/utilities/HttpClient'
+
+import * as authProvider from './AuthProvider'
+import * as loggerProvider from './LoggerProvider'
+import * as textProvider from './TextProvider'
+
+// as `backend` will always be accessed using `useRemoteBackend`.
```

Comment:
comment that was part of the old `@ts-expect-error` comment is no longer necessary